pandas/io/formats/info.py

```diff
@@ -622,7 +622,7 @@ def __init__(
     @property
     def max_rows(self) -> int:
         """Maximum info rows to be displayed."""
-        return get_option("display.max_info_rows", len(self.data) + 1)
```

Comment:
I'm confused as to what this was doing. Was it effectively passing `silent=True` always (since len >= 0)?

Reply:
Same here. I suspect this was a copy-paste + edit from a `set_option` with the wanted option possibly